src/app/components/quizList.tsx

```diff
@@ -0,0 +1,149 @@
+import Quiz, { QuizParams } from "@/app/components/quiz";
+import { useState } from "react";
+import clsx from "clsx";
+
+export interface QuizListParams {
+  className?: string;
+  quizzes: QuizParams[];
+  onFinish?: () => void;
```

Comment:
looks sus. what does the function do

Reply:
It will fire once all quizzes have been completed (in other words once you see your results). I'll add a comment